angular/pom.xml

```diff
@@ -23,6 +23,7 @@
     <artifactId>zeppelin</artifactId>
     <groupId>org.apache.zeppelin</groupId>
     <version>0.6.0-incubating-SNAPSHOT</version>
+    <relativePath>..</relativePath>
```

Comment:
@Leemoonsoo Yes ,you are right, it's safe to omit , explicitly defining is much clearer for reading and no more special reason,